ledger/participant-integration-api/src/main/scala/platform/store/cache/EventsBuffer.scala

```diff
@@ -118,28 +137,67 @@ class EventsBuffer[ENTRY](
       },
     )
 
-  /** Remove all buffered entries */
   def flush(): Unit = synchronized { _bufferLog = Vector.empty }
 }
 
-private[platform] object BufferSlice {
+private[cache] class EmptyEventsBuffer[E] extends EventsBuffer[E] {
+  override def push(offset: Offset, entry: E): Unit = ()
+
+  override def slice[FILTER_RESULT](
+      startExclusive: Offset,
+      endInclusive: Offset,
+      filter: E => Option[FILTER_RESULT],
+  ): BufferSlice[(Offset, FILTER_RESULT)] =
+    BufferSlice.LastBufferChunkSuffix(endInclusive, Vector.empty)
```

Comment:
Could name params here as an Exclusive param is being (correctly) passed an Inclusive variable.

Reply:
Right. Done 👍